blueprints/glimmer-helper/files/src/ui/components/__name__.ts

```diff
@@ -0,0 +1,2 @@
+export default function(params) {
```

Comment:
> shouldn't that be in src/ui/helpers?

We debated this for a while coming up with module unification, and settled on helpers and components in the same collection of invocable template functions.